### PR TITLE
Add template for getRuntime method

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: tests/acceptance/acceptance/TwigEnvironment.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@symfony-common
+Feature: LockableTrait
```
Comment (seferov) [inline tests/acceptance/acceptance/TwigEnvironment.feature:2]: it is not about lockabletrait

File: tests/acceptance/acceptance/TwigEnvironment.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    Background:
+        Given I have the following config
+      """
```
Comment (seferov) [inline tests/acceptance/acceptance/TwigEnvironment.feature:6]: indentations can be matched

File: tests/acceptance/acceptance/TwigEnvironment.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          <directory name="."/>
+          <ignoreFiles> <directory name="../../vendor"/> </ignoreFiles>
+        </projectFiles>
+        <plugins>
+          <pluginClass class="Psalm\SymfonyPsalmPlugin\Plugin"/>
+        </plugins>
+      </psalm>
+      """
+
+    Scenario: PropertyNotSetInConstructor error about $lock is not raised
```
Comment (seferov) [inline tests/acceptance/acceptance/TwigEnvironment.feature:19]: scenario also needs to fixed

**Review by seferov — CHANGES_REQUESTED** [review]

File: tests/acceptance/acceptance/TwigEnvironment.feature (hunk 2 above)
Comment (VincentLanglet) [reply tests/acceptance/acceptance/TwigEnvironment.feature]: Fixed, I also added an `.editorconfig` file.
This way the IDE is automatically using the same indentation than the project.

File: tests/acceptance/acceptance/TwigEnvironment.feature (hunk 1 above)
Comment (VincentLanglet) [reply tests/acceptance/acceptance/TwigEnvironment.feature]: Fixed

File: tests/acceptance/acceptance/TwigEnvironment.feature (hunk 3 above)
Comment (VincentLanglet) [reply tests/acceptance/acceptance/TwigEnvironment.feature]: Fixed

**Comment by seferov** [comment]
@VincentLanglet thank you!

**Pull request merged**